code/modules/client/preferences2/character_save.dm

```diff
@@ -471,6 +471,8 @@
 
 	character.dna.features = features.Copy()
 	character.set_species(chosen_species, icon_update = FALSE, pref_load = TRUE)
+	character.dna.features = features.Copy()
```

Comment:
This probably shouldnt be done twice. If set_species modifies DNA, do it after. If not then do it before

Reply:
Like this:
`if(character.set_species && ???)
		character.dna.features = features.Copy()
		character.set_species(chosen_species, icon_update = FALSE, pref_load = TRUE)
	else
		character.set_species(chosen_species, icon_update = FALSE, pref_load = TRUE)
		character.dna.features = features.Copy()
`

How do I set the if statement to tell if character.set_species has been modified?

Reply:
nevermind, turns out you only need the `character.dna.features = features.Copy()` after to work